data/selectors.json

```diff
@@ -15,5 +15,6 @@
   "storyTitle": ".mb-4",
   "tryAgain": ".btn",
   "storyImage": ".error__pic",
-  "storyText": ".card-text"
-}
+  "storyText": ".card-text",
+  "moral": "//p[contains(text(), 'Moral')]"
+ }
```

Comment:
Add an empty line at the end of the file

Reply:
done